provider/resource/configuration/v2/configuration.go

```diff
@@ -35,17 +35,16 @@ var RouteSchema *schema.Schema = &schema.Schema{
 			},
 			"telemetry_type": {
 				Type:        schema.TypeString,
-				Optional:    true,
+				Required:    true,
```

Comment:
This removes the `logs+metrics+traces` default value and requires the user to input `logs`, `metrics`, or `traces`.